src/Microsoft.AspNetCore.Server.Testing/Deployers/SelfHostDeployer.cs

```diff
@@ -74,8 +74,10 @@ protected CancellationToken StartSelfHost(Uri uri)
             }
             else
             {
+                var targetFramework = DeploymentParameters.TargetFramework ?? (DeploymentParameters.RuntimeFlavor == RuntimeFlavor.Clr ? "net451" : "netcoreapp1.0");
```

Comment:
I'm adding these default targets since I don't think they should change anymore and individual deployers can still override the default by specifying it explicitly. These defaults can also be added to `ApplicationDeployer`. The reason we had this option previously is because some apps contained multiple .NET Core targets (`dnxcore50` and `netstandardapp1.5` for example) making default targets less applicable but that is no longer the case.